RedditSharp/Things/LiveUpdateEvent.cs

```diff
@@ -545,7 +539,7 @@ private void CommonInit(Reddit reddit, JToken json, IWebAgent webAgent)
             WebAgent = webAgent;
         }
 
-        private void SimpleAction(string url)
+        private new void SimpleAction(string url)
```

Comment:
As mentioned in my new PR, it doesn't need to be protected as these special simple actions are only made in the LiveUpdate classes, and I don't really see why we should make them protected other than to not use the "new" clause, but I see no reason not to use the "new" clause otherwise